src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -1345,6 +1346,29 @@ var _ = Describe("installer HostRoleMaster role", func() {
 			Expect(errors.Is(err, context.DeadlineExceeded)).To(BeTrue())
 		})
 
+		It("service fail to sync - maxTimeout applied", func() {
+			WaitTimeout = 150 * time.Second
+			CVOMaxTimeout = 150 * time.Millisecond
+			currentServiceCVOStatus := &models.MonitoredOperator{Status: models.OperatorStatusProgressing, StatusInfo: ""}
+			clusterVersionReport := &configv1.ClusterVersion{
+				Status: configv1.ClusterVersionStatus{
+					Conditions: []configv1.ClusterOperatorStatusCondition{
+						{Type: configv1.OperatorAvailable, Status: configv1.ConditionTrue, Message: ""},
```

Comment:
I don't get it, this should pass no?
your clusterVersionReport available status is True

Reply:
this will get the same message all the time, you can see that we fail on timeout